src/phone-input/stateful-phone-input-container.js

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import React from 'react';
+import {COUNTRIES, STATE_CHANGE_TYPE} from './constants.js';
+
+import type {
+  StatefulPhoneInputContainerPropsT,
+  StateT,
+  StateReducerT,
+  StateChangeT,
+} from './types.js';
+import type {OnChangeParamsT} from '../select/types.js';
+
+const defaultStateReducer: StateReducerT = (type, nextState) => nextState;
+
+export default class StatefulPhoneInputContainer extends React.Component<
+  StatefulPhoneInputContainerPropsT,
+  StateT,
+> {
+  static defaultProps = {
+    initialState: {
+      text: '',
+      country: COUNTRIES.US,
+    },
+    onTextChange: () => {},
+    onCountryChange: () => {},
+    stateReducer: defaultStateReducer,
+    overrides: {},
+  };
+
+  state = {...this.props.initialState};
```

Comment:
If we do `state = {text: '', country: COUNTRIES.US, ...this.props.initialState}` then we won't require the both `text` and `country` always to be passed together.

Reply:
ah good point, i will update